### PR TITLE
Update trigger_build.sh to use CircleCI API v2

### PR DESCRIPTION
**Pull request by matiasgarcia**
Fixes https://github.com/bitex-la/storyboard/issues/1395

### CODE REVIEW

**Comment by matiasgarcia** [comment]
@mdallago CI: https://app.circleci.com/pipelines/github/bitex-la/compliance?branch=update-trigger-build-to-use-circle-api-v2 triggered https://app.circleci.com/pipelines/github/bitex-la/cryptocambios?branch=master

File: .circleci/trigger_build.sh [hunk 1]
```diff
@@ -4,9 +4,11 @@ printf "Triggering an $PROJECT_TO_BUILD build on the $CIRCLE_BRANCH branch\n\n"
 
 if [ "$CIRCLE_BRANCH" = "master" ] 
 then
-  BUILD_INFO=$(curl -X POST -H -d \
-    "{}" \
-    "https://circleci.com/api/v1/project/$ORGANIZATION/$PROJECT_TO_BUILD/tree/$CIRCLE_BRANCH?circle-token=$CIRCLE_TOKEN")
+  BUILD_INFO=$(curl --request POST \
+  --url "https://circleci.com/api/v2/project/github/$ORGANIZATION/$PROJECT_TO_BUILD/pipeline" \
+  --header 'Circle-Token: aae23a3f48ce2b01bcf361f2820411aafb67a7c8' \
```
Comment (mdallago) [inline .circleci/trigger_build.sh]: te quedo hardcoding el token

File: .circleci/trigger_build.sh [hunk 2]
```diff
@@ -4,9 +4,11 @@ printf "Triggering an $PROJECT_TO_BUILD build on the $CIRCLE_BRANCH branch\n\n"
...
+  --header 'content-type: application/json' \
+  --data '{"branch":"master"}')
```
Comment (mdallago) [inline .circleci/trigger_build.sh]: quedo hardcoding el branch

**Review by mdallago — CHANGES_REQUESTED** [review]
ver comments

**Review by mdallago — CHANGES_REQUESTED** [review]
este archivo no va config/storage.yml

**Review by mdallago — APPROVED** [review]

**Pull request merged**